content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Building your own Collector instance"
+weight: 30
+---
+
+If you are planning to build and debug your components, you are going to need your own Collector instance. That will allow you to launch and debug your OpenTelemetry Collector components directly within your favorite Golang IDE.
+
+The other interesting aspect of approaching the component development this way is that you can use all the debugging features from your IDE (stack traces are great teachers!) to understand how the Collector itself interacts with your component code.
+
+In order to make it easy for developers to create their own custom builds of the Collector, the OpenTelemetry Community has developed a tool called [OpenTelemetry Collector builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder).
+
+The `builder` is meant to help developers to quickly assemble and build their own Collector's distributions, based on a given configuration file.
```

Comment:
That's not the main goal for the builder. The builder exists to assist people in assembling their own distribution, making it easy to build a distribution that includes their custom components along with components that are publicly available.

The developer use-case is a nice added benefit though :-)

Reply:
@jpkrohling fair enough, let me fix that.

Reply:
fixed in: https://github.com/open-telemetry/opentelemetry.io/pull/1277#:~:text=to%20support%20suggestions.-,fd2e9a3,-Merge%20branch%20%27doc